README.md

```diff
@@ -8,7 +8,70 @@ assisted close reading tool
 
 [https://github.com/ADAH-EviDENce/evidence-gui](https://github.com/ADAH-EviDENce/evidence-gui)
 
+## Running a demo on Windows
 
+Prerequisites:
+- Docker desktop
+
```

Comment:
How would users test the Docker installation? Maybe test with hellow-world image?

Reply:
Added docker run hello-world instructions